app/components/primer/button_component.rb

```diff
@@ -56,6 +56,21 @@ class ButtonComponent < Primer::Component
     }
     alias counter trailing_visual_counter # remove alias when all buttons are migrated to new slot names
 
+    # `Tooltip` that appears on mouse hover or keyboard focus over the button. Use tooltips sparingly and as a last resort.
+    # Consult the <%= link_to_component(Primer::Alpha::Tooltip) %> documentation for more information.
```

Comment:
I think we may want to call extra ttention to the `type:` here. Maybe something like:

```suggestion
    # **Important:** This tooltip defaults to `type: :description`. In a few scenarios, `type: :label` may be more appropriate.
    # Consult the <%= link_to_component(Primer::Alpha::Tooltip) %> documentation for more information.
```